Makefile

```diff
@@ -46,6 +48,7 @@ DEFAULT_EPSG_EXTEND ?= '[420000, 30000, 900000, 350000]'
 DEFAULT_ELEVATION_MODEL ?= COMB
 DEFAULT_TERRAIN ?= ch.swisstopo.terrain.3d
 SAUCELABS_TESTS ?=
+USERNAME := $(shell whoami)
```

Comment:
Indeed, a good idea. User specific is default, not main, as it is now.
I'll adapt.

On Thu Mar 17 15:38:10 2016 GMT+0100, Gasser Loïc wrote:
> > @@ -46,6 +48,7 @@ DEFAULT_EPSG_EXTEND ?= '[420000, 30000, 900000, 350000]'
> >  DEFAULT_ELEVATION_MODEL ?= COMB
> >  DEFAULT_TERRAIN ?= ch.swisstopo.terrain.3d
> >  SAUCELABS_TESTS ?=
> > +USERNAME := $(shell whoami)
> 
> @procrastinatio suggested to me that we could a single rc_user file doing:
> 
> export USERNAME='$(shell whoami)'                                               
> export API_URL=//mf-chsdi3.dev.bgdi.ch/$USERNAME                                
> export APACHE_BASE_PATH=/$USERNAME                                              
> export E2E_TARGETURL=https://mf-geoadmin3.dev.bgdi.ch/$USERNAME
> 
> -> we would the ignore all the other rc files that are user specific
> 
> I would simply add in makefile:
> USER_SOURCE ?= rc_user
> 
> so that you can override this behaviour using:
> 
> `make user USER_SOURCE=rc_ltgal`
> 
> or simply
> 
> `source rc_ltgal && make all`
> 
> like before
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/geoadmin/mf-geoadmin3/pull/3229/files/8356402aa3abd0d785d4fb4dd303b2f3f69b0b5e#r5651404